server/controllers/importers/graphml.js

```diff
@@ -1,19 +1,84 @@
+
 var constants = require(__dirname + "/../constants");
 var parseString = require("xml2js").parseString;
 var semanticChecker = require(__dirname + "/../semantic-checker");
 
+var graphmlWarnings = {
+  EDGES_WITHOUT_WEIGHTS: {
+    warningCode: "EDGES_WITHOUT_WEIGHTS",
+    errorDescription: "GRNsight has detected that one or more edges in your network are missing numerical weight" +
+    " values. Because the algorithm GRNsight uses for determining the arrowhead type and the color and thickness of" +
+    " the edges requires numerical weight values, your graph will display as an unweighted graph with black edges" +
+    " and pointed arrowheads. If you want to display the network as a weighted graph, please modify your input file" +
+    " to include weight values for all edges."
```

Comment:
Indent unfinished expressions, to show that those lines are “under” the `errorDescription` property name.